TAC/Meetings/2025/2025-05-15/CCC_TAC_Minutes-2025-05-15

```diff
@@ -0,0 +1,118 @@
+# Confidential Computing Consortium
+
+## Minutes of the Technical Advisory Council
+
+*The TAC meets for two hours on alternating Thursdays. All members of the community are welcome to attend and participate.*
+
+### Meeting details
+
+**Date**: Thursday, May 15, 2025
+
+**Time**: 7a-9a US Pacific Time
+
+* **Zoom**: [https://zoom-lfx.platform.linuxfoundation.org/meeting/94618773737?password=4b2a5cdf-685a-4ea3-822d-24ff7ddab72e) 
+
+* **Calendar**: [https://calendar.confidentialcomputing.io](https://calendar.confidentialcomputing.io),
+[ical](https://calendar.google.com/calendar/ical/c\_c0pcihr7n2n1k3a38i32d9ag10%40group.calendar.google.com/public/basic.ics),
+[Google Calendar](https://calendar.google.com/calendar/u/0/r?cid=c\_c0pcihr7n2n1k3a38i32d9ag10@group.calendar.google.com)
+
+* **Recordings**: [YouTube TAC Playlist](https://www.youtube.com/playlist?list=PLmfkUJc39uMjaB_I1dYW72I44kr9QzG_B)
+
+## Links
+
+* **Code of Conduct**: [code-of-conduct.confidentialcomputing.io](https://code-of-conduct.confidentialcomputing.io)
+* **CCC Charter**: [charter.confidentialcomputing.io](https://charter.confidentialcomputing.io)
+* **LF Training course on DEI**: [Inclusive Open Source Community Orientation (LFC102) (free)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
+* **Declared project dependencies**: [Google Sheets](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0)
+* **CCC YouTube**: [youtube.confidentialcomputing.io](https://youtube.confidentialcomputing.io)
+* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
+* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+## Agenda and Minutes
+
+### Welcome
+
+* Dan Middleton (DM) opened the call at 7:03 am PT
+* DM welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation
+* Renu Chauhan (RC) & Riaan Kleinhans (RK) created meeting minutes.
+* DM reviewed the agenda
+
+
+### Roll Call / Attendance
+
+A quorum were not required
```

Comment:
We did establish quorum to formalize moving Veracruz to Emeritus.

Reply:
Updated